root/user/ReportUser.js

```diff
@@ -0,0 +1,158 @@
+      value: 'spam',
+    },
+    {
+      label: N_l('Editor is unresponsive to edit notes'),
+      value: 'unresponsiveness',
+    },
+    {
+      label: N_l('Editor intentionally ignores accepted guidelines'),
+      value: 'ignoring_guidelines',
+    },
+    {
+      label: N_l('Editor is overzealous in enforcing guidelines as rules'),
+      value: 'enforcing_guidelines',
+    },
+    {
+      label: N_l('Editor engages in overzealous or abusive yes/no voting'),
+      value: 'voting',
+    },
+    {
+      label: N_l('Editor has violated some other part of our Code of Conduct'),
+      value: 'other',
+    },
+  ],
+};
+
+const ReportUser = ({
+  $c,
+  form,
+  user,
+}: Props) => {
+  form.field.reveal_address.value = true;
+  return (
+    <UserAccountLayout
+      page="report"
+      title={l('Report user')}
+      user={user}
+    >
+      <h2>{l('Report user')}</h2>
+
+      <p>
+        {l('Please review our {uri|Code of Conduct} before sending a report.',
+          {
+            uri: {
+              href: '/doc/Code_of_Conduct',
+              target: '_blank',
+            },
+          })}
+      </p>
+
+      <p>
+        {l(`Your report will be sent to our {uri|account administrators},
+           who will decide what action to take.`,
+        {
+          uri: {
+            href: '/privileged',
+            target: '_blank',
+          },
+        })}
+      </p>
+
+      <p>
+        {l(`If you’d like us to be able to easily respond to your report,
+          please check “Reveal my email address” below.`)}
+      </p>
+
+      <p>
+        <strong>{addColon(l('Note'))}</strong>
+        {' '}
+        {l(`Be sure to provide direct links to examples of the behaviour you’re
+          reporting (for example, use “https://musicbrainz.org/edit/23”
+          instead of “edit #23”, and use “https://musicbrainz.org/edit/42 and
+          https://musicbrainz.org/edit/43” instead of
+          “https://musicbrainz.org/user/SomeUser/edits”).
+          Providing links makes it much easier for the recipients of the
+          report to look into the issues; a report without links is
+          unlikely to be acted on fast, since it will require a lot of
+          additional research.`)}
+      </p>
+
+      <form action={$c.req.uri} className="report-form" method="post">
+        <FormRowSelect
+          field={form.field.reason}
+          label={addColon(l('Reason'))}
+          onChange={noop}
+          options={reportReasonOptions}
+        />
+
+        <FormRowTextArea
+          cols={50}
+          field={form.field.message}
+          label={addColon(l('Message'))}
+          required
+          rows={10}
+        />
+
+        <FormRow hasNoLabel>
+          <p>
+            {l(`If you don't want our admins to contact you further regarding
+                this report, you can uncheck the checkbox below.`)}
+            <br />
+            {l(`We recommend leaving it checked, so that you can be contacted
```

Comment:
Would it make sense to combine these two strings + the `<br />` into one? The second kinda follows directly from the first.

Reply:
I was trying to avoid having the <br /> in the string, but I guess that'd make sense? Done